src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/config/ResourceServerConfiguration.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsauditapi.config
+
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
+import org.springframework.security.config.http.SessionCreationPolicy
+
+@Configuration
+@EnableWebSecurity
+@EnableGlobalMethodSecurity(prePostEnabled = true, proxyTargetClass = true)
+class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
+  override fun configure(http: HttpSecurity) {
+    http
+      .sessionManagement()
+      .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+      .and().csrf().disable()
+      .authorizeRequests { auth ->
+        auth.antMatchers(
+          "/webjars/**", "/favicon.ico", "/csrf",
```

Comment:
Not sure if I need all these. There is no UI so probably not?

Reply:
I tried to remove webjars once - might have broken the swagger docs?  Try it and see :)